src/simulator/simulator.cpp

```diff
@@ -100,17 +100,15 @@ void Simulator::init(ros::NodeHandle& n) {
   odometryTwistMsg.header.frame_id = "odom";
   odometryTwistMsg.child_frame_id = "base_footprint";
 
-  curLoc = Vector2f(cStartX, cStartY);
-  curAngle = cStartAngle;
+  cur_loc = Vector2f(cStartX, cStartY);
+  cur_angle = cStartAngle;
 
+  // TODO(jaholtz): Determine what motion model to use based on config file
+  motion_model_ = new ackermann::AckermannModel("config/f1_config.lua", n);
```

Comment:
When you make `motion_model_` a `std::unique_ptr`, you will need to call something like `std::make_unique`.